pkg/ddl/constraint_test.go

```diff
@@ -287,3 +287,21 @@ func TestAlterEnforcedConstraintStateChange(t *testing.T) {
 	tk.MustExec("alter table t alter constraint c1 enforced")
 	tk.MustQuery("select * from t").Check(testkit.Rows("12"))
 }
+
+// Related issue TiDB#47567, #47631 and #47632.
+func TestCheckConstraintIssue47567(t *testing.T) {
```

Comment:
We can move this test case to https://github.com/pingcap/tidb/blob/master/tests/integrationtest/t/ddl/constraint.test#L660, and run it by 
```bash
cd tests/integrationtest
bash ./run-tests.sh -s ../../bin/tidb-server -r ddl/constraint
```

Reply:
Tests have been moved.